src/apiCalls.js

```diff
@@ -0,0 +1,14 @@
+
+export const getPlaces = (url) => {
+    return fetch(url)
+      .then(res => {
+        if (!res.ok) {
+          throw new Error('Oops! We seem to be having some technical issues, please try again later!');
+        }
+        return res.json();
+      })
+      .catch(error => {
```

Comment:
Totally optional, and I should have mentioned this the first time. I suspect the catch block on line 10 is not necessary since you have the catch in the App component.

Reply:
@Pma913 oh that's interesting. I would have written it how Houda has it, too. How about we leave it for now and compare the user flow with vs. without the catch block to make sure it's not needed?

Reply:
Sounds good to me. Let's merge it!